### PR TITLE
NETOBSERV-690: move console and FLP image pullspecs from CRD to CSV

### PR DESCRIPTION
**Pull request by mariomac**
Continuation of PR #690. Now the related images (eBPF agent, FLP and console plugin) aren't specified anymore as part of the FlowCollector spec. If you want to deploy a development image for these components, you should now patch the environment variables of the `netobserv-controller-manager` Deployment, as specified in the DEVELOPERS.md section (see the diff of this PR).

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -233,16 +233,14 @@ endif
...
 	cd config/manager && $(KUSTOMIZE) edit set image controller=$(IMG)
 	$(SED) -i -r 's~ebpf-agent:main~ebpf-agent:$(BPF_VERSION)~' ./config/manager/manager.yaml
-	cp config/samples/flows_v1alpha1_flowcollector.yaml config/samples/flows_v1alpha1_flowcollector_versioned.yaml
-	$(SED) -i -r 's~flowlogs-pipeline:main~flowlogs-pipeline:$(FLP_VERSION)~' config/samples/flows_v1alpha1_flowcollector_versioned.yaml
-	$(SED) -i -r 's~console-plugin:main~console-plugin:$(PLG_VERSION)~' config/samples/flows_v1alpha1_flowcollector_versioned.yaml
+	$(SED) -i -r 's~flowlogs-pipeline:main~flowlogs-pipeline:$(FLP_VERSION)~' ./config/manager/manager.yaml
+	$(SED) -i -r 's~console-plugin:main~console-plugin:$(PLG_VERSION)~' ./config/manager/manager.yaml
 	$(SED) -i -r 's~blob/[0-9]+\.[0-9]+\.[0-9]+(-rc[0-9]+)\?/~blob/$(VERSION)/~g' ./config/manifests/bases/netobserv-operator.clusterserviceversion.yaml
 	$(SED) -i -r 's~replaces: netobserv-operator\.v.*~replaces: netobserv-operator\.$(PREVIOUS_VERSION)~' ./config/manifests/bases/netobserv-operator.clusterserviceversion.yaml
+	cp config/samples/flows_v1alpha1_flowcollector.yaml config/samples/flows_v1alpha1_flowcollector_versioned.yaml
```
Comment (mariomac) [inline Makefile:243]: actually the `flows_v1alpha1_flowcollector_versioned.yaml` file has no sense anymore. Shall we remove it in a future PR?

Comment (jpinsonneau) [reply Makefile:243]: for me yes

**Review by jpinsonneau — APPROVED** [review]
LGTM in terms of code

**Review by OlivierCazade — APPROVED** [review]
LGTM, thanks!

**Pull request merged**